compiler/rustc_hir_typeck/src/upvar.rs

```diff
@@ -2093,7 +2093,7 @@ fn restrict_precision_for_unsafe(
     }
 
     for (i, proj) in place.projections.iter().enumerate() {
-        if proj.ty.is_unsafe_ptr() {
+        if proj.ty.is_raw_ptr() {
             // Don't apply any projections on top of an unsafe ptr.
```

Comment:
Please also grep the codebase for "unsafe ptr" and "unsafe pointer" and update what you can find (except for the one in `RELEASES.md`, let's not change historic data like old release notes).

Reply:
Good point, I tried to do that already but seems like I missed this. I extend it now.